projects/packages/my-jetpack/_inc/components/my-jetpack-screen/styles.module.scss

```diff
@@ -1,26 +1,38 @@
+@import '../../../node_modules/@wordpress/base-styles/breakpoints';
+@import '../../../node_modules/@wordpress/base-styles/mixins';
+
 .heading {
 	margin-bottom: 0;
 }
 
 .notice {
+	box-sizing: border-box;
 	margin: 0;
+	padding: 12px 20px 12px 14px;
 	font-size: 16px;
+	line-height: 22px;
```

Comment:
We should be using `Text` component at this one, instead of manually inserting `line-height` and `font-size`.